src/utils/CalendarUtils/GitHub/GitHub.js

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable no-console */
+
+import { parseSync } from 'svgson';
+import * as Proxy from '../../Proxy/Proxy';
+import * as JavaScriptUtils from '../../JavaScriptUtils/JavaScriptUtils';
+
+const getCurrentUserSvg = async (proxyServerUrl, gitHubUsername) => {
```

Comment:
Why **current**User when the username is passed?

Reply:
Yes, that's a mistake. Actually, I copy-pasted the lines from the other repository.

I'll rename the function to `getUserSvg`.